test/LintStyle.lean

```diff
@@ -135,3 +135,18 @@ lemma foo' : True := trivial
 -- TODO: add terms for the term form
 
 end setOption
+
+section unicodeLinter
+
+open Mathlib.Linter.TextBased
+
+#guard let errContext : ErrorContext := {
+    error := .unwantedUnicode 'Z', lineNumber := 4, path:="./MYFILE.lean"}
+  (parse?_errorContext <| outputMessage errContext .exceptionsFile) == some errContext
+
+-- The list `othersInMathlib` should only contain characters not present in another list.
+#guard unicodeLinter.othersInMathlib.toList ∩ unicodeLinter.withVSCodeAbbrev.toList = ∅
+#guard unicodeLinter.othersInMathlib.toList ∩ unicodeLinter.emojis.toList = ∅
+#guard unicodeLinter.othersInMathlib.toList ∩ unicodeLinter.nonEmojis.toList = ∅
```

Comment:
Maybe we should add some more here? Intersection of `emojis` and `nonEmojis` for example?

Reply:
I added some.